pkg/ccl/backupccl/backup_destination_test.go

```diff
@@ -238,10 +225,10 @@ func TestBackupRestoreResolveDestination(t *testing.T) {
 				{
 					to := localizeURI(t, baseDir, localities)
 					// The full backup should go into the baseDir.
```

Comment:
Discussed in slack - this doesn't get us much from the UX perspective, but the code is _probably_ cleaner since we don't have to make copies and fork more on syntax type. Your thoughts?

Reply:
just read through the slack thread. I'm ok with this change! To clarify for Kathryn and future users, I'd amend the commit message so it's clear that this change modifies both `BACKUP TO` and `BACKUP INTO`.